workshop/content/intermediate/templates/dynamic-references/_index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: 'Dynamic References'
+date: 2022-02-02T20:33:04Z
+weight: 300
+---
+
+### Overview
+
+In this module, you will learn how to use [dynamic references](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html) in your CloudFormation template to reference external values stored in other AWS services such as [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html)  (SSM) and [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html).
+
+When you organize your AWS resources based on lifecycle and ownership, you might want to build a stack that references configuration values that are central to multiple stacks. The AWS Systems Manager Parameter Store provides secure, hierarchical storage for configuration data management.
+
+In other cases, you might be required to reference sensitive information in your AWS CloudFormation templates. [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) helps you to securely encrypt, store, and retrieve credentials for your databases and other services programmatically.
```

Comment:
SSM Secure is one of the supported string types in Systems Manager Parameter Store. I'm not sure if this would be the right place to discuss about a distinct string type.

Reply:
I have added a suggestion - please have a look!  Thanks!